oecophylla/cli/tests/data/exp_config.yaml

```diff
@@ -12,18 +12,21 @@ params:
   abundance_samples:
   - sample_S22282
   - sample_S22205
-  anvio:
-    centrifuge_base: /home/jgsanders/miniconda/envs/anvio2/centrifuge
-    centrifuge_models: /home/jgsanders/miniconda/envs/anvio2/centrifuge/b+h+v/b+h+v
   assemblers:
   - megahit
   - metaspades
-  atropos: ' -a GATCGGAAGAGCACACGTCTGAACTCCAGTCAC -A GATCGGAAGAGCGTCGTGTAGGGAAAGGAGTGT
-    -q 15 --minimum-length 100 --pair-filter any'
   binning_samples:
   - sample_S22282
   - sample_S22205
-  filter_db: test_data/phix
+  mapping_assembler:
+  - metaspades
+  filter_db: test_data/test_dbs/bowtie2/phix
+  trimmer: atropos
+  anvio:
+    centrifuge_base: /home/jgsanders/miniconda/envs/anvio2/centrifuge
```

Comment:
are we OK pointing to your home dir?

Reply:
These paths are still legacy and we'll update with the Anvi'o module...